index.bs

```diff
@@ -1162,6 +1162,7 @@ dictionary SaveFilePickerOptions : FilePickerOptions {
 };
 
 dictionary DirectoryPickerOptions {
+  boolean requestWritable = false;
```

Comment:
per discussion offline, what about changing this to `FileSystemPermissionMode mode = "read"`?

Reply:
This has been updated.